Sources/StreamChatUI/ChatMessageList/ChatChannelHeaderView.swift

```diff
@@ -127,7 +127,7 @@ open class ChatChannelHeaderView:
 
     open func channelController(
         _ channelController: ChatChannelController,
-        didReceiveMemberEvent: MemberEvent
+        didReceiveMemberEvent: Event
```

Comment:
why this change?

Reply:
`MemberEvent` needs to have `savedData`, which is defined in `EventWithPayload`. `EventWithPayload` is `internal`, so we either expose `EventWithPayload` (bad idea) or we change these params `MemberEvent` -> `Event` so we don't need to expose payloads. The good part is, this change is basically free for the user, since the user needs to cast `MemberEvent` to a specific event (`MemberAddedEvent` or `MemberRemovedEvent`) anyway